modules/virt-configuring-vm-project-dpdk.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-project-dpdk_{context}"]
+= Configuring a project for DPDK workloads
+
+Configure the project to run DPDK workloads on SR-IOV hardware. 
+
+.Prerequisites
+* Your cluster is configured to run DPDK workloads.
+
+.Procedure
+. Create a namespace for your DPDK applications:
++
+[source,terminal]
+----
+$ oc create ns dpdk-checkup-ns
+----
+
+. Create an `SriovNetwork` object that references the `SriovNetworkNodePolicy` object. When you create an `SriovNetwork` object, the SR-IOV Network Operator automatically creates a `NetworkAttachmentDefinition` object.
++
+.Example `SriovNetwork` manifest
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: dpdk-sriovnetwork
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |
+    {
+      "type": "host-local",
+      "subnet": "10.56.217.0/24",
+      "rangeStart": "10.56.217.171",
+      "rangeEnd": "10.56.217.181",
+      "routes": [{
+        "dst": "0.0.0.0/0"
+      }],
+      "gateway": "10.56.217.1"
+    }
+  networkNamespace: dpdk-checkup-ns
```

Comment:
Please consider adding a `<x>` here to explain that this is the namespace you want to deploy the NetworkAttachmentDefinition object to.